django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -45,6 +45,7 @@ def gis_operators(self):
             "bboverlaps": SpatialOperator(func="MBROverlaps"),  # ...
             "contained": SpatialOperator(func="MBRWithin"),  # ...
             "contains": SpatialOperator(func="ST_Contains"),
+            "coveredby": SpatialOperator(func="MBRCoveredBy"),
```

Comment:
It seems that there is limited coverage for this operator. It seems that something like this would still result in the test suite passing. 

```suggestion
            "coveredby": SpatialOperator(func="ST_Contains"),
```

If you think it's worth it maybe we could add an extra test. Here's something I drafted. It's my first time testing QuerySets so I'm sure this isn't best practice. 

``` python
    @skipUnlessGISLookup("coveredby")
    def test_coveredby_lookup(self):
        poly = Polygon(LinearRing((0, 0), (0, 5), (5, 5), (5, 0), (0, 0)))
        State.objects.create(name="Test", poly=poly)
        qs = State.objects.filter(poly__coveredby=poly)
        self.assertQuerySetEqual(qs, [poly.wkt], attrgetter("poly.wkt"))

        small_poly = Polygon(LinearRing((1, 1), (1, 4), (4, 4), (4, 1), (1, 1)))
        qs = State.objects.filter(poly__coveredby=small_poly)
        self.assertQuerySetEqual(qs, [])

        large_poly = Polygon(LinearRing((-1, -1), (-1, 6), (6, 6), (6, -1), (-1, -1)))
        qs = State.objects.filter(poly__coveredby=large_poly)
        self.assertQuerySetEqual(qs, [poly.wkt], attrgetter("poly.wkt"))
```

Reply:
Good point :eagle:

Reply:
It seems like this never worked on Oracle :smiling_face_with_tear:

Reply:
I adjusted example to pass on Oracle:

- `COVERS`: _The boundary and interior of one object is completely contained in the interior or the boundary of the other object, their interiors intersect, and the boundary or the interior of one object and the boundary of the other object intersect._ 

- `COVEREDBY`: _The opposite of `COVERS`. A `COVEREDBY` B implies B `COVERS` A._


https://docs.oracle.com/en/database/oracle/oracle-database/23/spatl/spatial-relationships-and-filtering.html#GUID-FA71CB03-FFF0-4529-BA07-AD0605DA7A89

so boundaries must intersect, and objects cannot be equal.